backend/de.metas.cucumber/src/test/resources/de/metas/cucumber/features/contract/moduleLog/processed/salesProcessedProduct.feature

```diff
@@ -0,0 +1,567 @@
+
+    And validate M_Shipping_NotificationLines:
+      | M_Shipping_NotificationLine_ID.Identifier | M_Shipping_Notification_ID.Identifier | M_ShipmentSchedule_ID.Identifier | M_Product_ID.Identifier | MovementQty |
+      | shippingNotificationLine_03102024_1       | shippingNotification_03102024_1       | sch_1                            | processedProduct        | 1000        |
+
+    And after not more than 30s, ModCntr_Logs are found:
+      | ModCntr_Log_ID.Identifier | Record_ID.Identifier                | ContractType    | OPT.CollectionPoint_BPartner_ID.Identifier | OPT.M_Warehouse_ID.Identifier | M_Product_ID.Identifier | OPT.Producer_BPartner_ID.Identifier | OPT.Bill_BPartner_ID.Identifier | Qty  | TableName                   | C_Flatrate_Term_ID.Identifier | ModCntr_Type_ID.Identifier | OPT.Processed | OPT.ModCntr_Log_DocumentType | OPT.C_Currency_ID.ISO_Code | OPT.C_UOM_ID.X12DE355 | OPT.Amount | OPT.Harvesting_Year_ID.Identifier | OPT.ModCntr_Module_ID.Identifier | OPT.PriceActual | OPT.Price_UOM_ID.X12DE355 | OPT.ProductName              | OPT.IsBillable |
+      | log_avInterim_snline      | shippingNotificationLine_03102024_1 | ModularContract | bp_moduleLogPO                             | warehouse_03102024_1          | addValueOnInterim       | bp_moduleLogPO                      | bp_moduleLogPO                  | 1000 | M_Shipping_NotificationLine | moduleLogContract_1           | avInterim_type             | false         | ShippingNotification         | EUR                        | PCE                   | -20000     | y2022                             | PO_avOnInterim_module            | -20.00          | PCE                       | addValueOnInterim_03102024_1 | Y              |
+      | log_svInterim_snline      | shippingNotificationLine_03102024_1 | ModularContract | bp_moduleLogPO                             | warehouse_03102024_1          | subValueOnInterim       | bp_moduleLogPO                      | bp_moduleLogPO                  | 1000 | M_Shipping_NotificationLine | moduleLogContract_1           | svInterim_type             | false         | ShippingNotification         | EUR                        | PCE                   | 20000      | y2022                             | PO_svOnInterim_module            | 20.00           | PCE                       | subValueOnInterim_03102024_1 | Y              |
+
+    And the shipment schedule is split
+      | Identifier | M_ShipmentSchedule_ID.Identifier | QtyToDeliver | C_UOM_ID.X12DE355 | DeliveryDate | OPT.UserElementNumber1 | OPT.UserElementNumber2 |
+      | spl_1      | sch_1                            | 750          | PCE               | 2022-04-04   | 700                    | 1                      |
+      | spl_2      | sch_1                            | 250          | PCE               | 2022-04-04   | 1200                   | 5                      |
```

Comment:
it would be nice to have atleast 60d between physical clearance date and deliverydate to properly test 30d per month rule + freeStorageDays

Reply:
changed to >60